src/components/content/_content.scss

```diff
@@ -0,0 +1,46 @@
+.content {
+  &__heading-2 {
+    color: var(--color-heading-2);
+    font-size: $font-size-h2;
+
+    @media screen and (min-width: map-get($breakpoints, 'breakpoint-3')) {
+      font-size: 2.2rem;
```

Comment:
This is the second time I've seen `font-size: 2.2rem`. Should that be a standard size, moved into a variable?